aws/sagemaker-pytorch-container
# [WIP] change: add buildspec-release file

*Issue #, if available:*

*Description of changes:*
- fix tox file
- use tox instead of pytest in buildspec
- add cpu_py3 and gpu_py2 tests in buildspec
- add buildspec-release file

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
